### ISSUE TITLE
VarLenQField Example Broken

### ISSUE THREAD

**Issue by dhankook** [secdev/scapy#1301]
At [http://scapy.readthedocs.io/en/latest/build_dissect.html#example-variable-length-quantities](http://scapy.readthedocs.io/en/latest/build_dissect.html#example-variable-length-quantities), class `VarLenQField` is defined as a subclass of `Field`.

Any attempt to instantiate VarLenQField results in:
`AttributeError: 'VarLenQField' object has no attribute 'fld'`

The offending line is:
`self.fld = fld`

Somehow, inheriting from `Field` is causing the problem.  The error happens whether in the scapy iPython environment or executing a python script.

**Comment by guedou** [comment]
Good catch! This definition is now incorrect and we did not update the documentation, the `fld` field must be defined in the `__slots__` list:
```
class VarLenQField(Field):
    """ variable length quantities """
    __slots__ = ["fld"]
    
    def __init__(self, name, default, fld):
        Field.__init__(self, name, default)
        self.fld = fld 
```

Can you provide a pull-request?

**Comment by dhankook** [comment]
I have done so, my first ever, so let me know if I didn't do it correctly.  Thanks

**Comment by guedou** [comment]
Fixed byt #1302
